zepto-dnd.js

```diff
@@ -661,6 +730,9 @@
   }
 
   Sortable.prototype.over = function(e) {
+    if (this.scroller)
+      this.scroller.scrollEvent(this.el, e);
+
```

Comment:
Thanks.  I tested and it worked fine without those extra lines.

I've pushed my amendments anyway since I didn't read your note.